pkg/cli/flags.go

```diff
@@ -657,6 +658,7 @@ func init() {
 	// The --empty flag is only valid for the top level demo command,
 	// so we use the regular flag set.
 	BoolFlag(demoCmd.Flags(), &demoCtx.useEmptyDatabase, cliflags.UseEmptyDatabase, demoCtx.useEmptyDatabase)
+	StringFlag(demoFlags, &demoCtx.geosLocation, cliflags.GEOSLocation, "")
```

Comment:
ditto (Although maybe you could share the same field `startCtx.geosLocation` for both commands.)